urdf2webots/writeRobot.py

```diff
@@ -449,7 +449,7 @@ def URDFVisual(robotFile, visualNode, level, normal=False):
                 robotFile.write((shapeLevel + 2) * indent + '}\n')
 
                 robotFile.write((shapeLevel + 2) * indent + 'textureTransform TextureTransform {\n')
-                robotFile.write((shapeLevel + 3) * indent + 'scale 1 -1\n')
+                robotFile.write((shapeLevel + 3) * indent + 'scale 1 1\n')
```

Comment:
As `scale` has the default value, a better fix would be to remove the three lines adding the `textureTransform` node.

Reply:
Well @omichel has been faster than me.

Reply:
But your point is better than mine as the `TextureTransform` node is actually useless.